bundles/org.openhab.binding.benqprojector/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,173 @@
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="projector-serial">
+		<label>BenQ Projector - Serial</label>
+		<description>A BenQ projector connected via a serial port</description>
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="picturemode" typeId="picturemode"/>
+			<channel id="aspectratio" typeId="aspectratio"/>
+			<channel id="freeze" typeId="freeze"/>
+			<channel id="blank" typeId="blank"/>
+			<channel id="directcmd" typeId="directcmd"/>
+			<channel id="lamptime" typeId="lamptime"/>
+		</channels>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<label>Serial Port</label>
+				<context>serial-port</context>
+				<description>Serial Port to Use for Connecting to the BenQ Projector</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" min="5" max="60" unit="s" required="false">
+				<label>Polling interval</label>
+				<description>Configures How Often to Poll the Projector for Updates (5-60; Default 10)</description>
+				<default>10</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="projector-tcp">
+		<label>BenQ Projector - TCP/IP</label>
+		<description>A BenQ projector connected via the built-in ethernet port or a serial over
+			IP device</description>
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="picturemode" typeId="picturemode"/>
+			<channel id="aspectratio" typeId="aspectratio"/>
+			<channel id="freeze" typeId="freeze"/>
+			<channel id="blank" typeId="blank"/>
+			<channel id="directcmd" typeId="directcmd"/>
+			<channel id="lamptime" typeId="lamptime"/>
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<context>network-address</context>
+				<description>IP address for the projector or serial over IP device</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="true">
+				<label>Port</label>
+				<description>Port for the projector or serial over IP device</description>
+				<default>8000</default>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" min="5" max="60" unit="s" required="false">
+				<label>Polling interval</label>
+				<description>Configures How Often to Poll the Projector for Updates (5-60; Default 10)</description>
+				<default>10</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Powers the Projector On or Off</description>
+	</channel-type>
```

Comment:
You could reuse the predefined system channel, instead of defining your own. https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Reply:
done